### PR TITLE
*: refactor logging to support custom settings for different session

### PR DESCRIPTION
**Pull request by AstroProfundis**
### What problem does this PR solve? 
Using of package wide logger is not friendly to other packages to import, if any of the tiup code run inside goroutines they all have to use the same logging settings.

### What is changed and how it works?
Set a logger for each task and manager, so that multiple instances can use different loggers.

### Check List 

Tests 

 - Integration test
 - Manual test (add detailed scripts or steps below)

Code changes

 - Has exported function/method change
 - Has exported variable/fields change

Side effects

 - Increased code complexity

Release notes:

```release-note
NONE
```

### CODE REVIEW

File: pkg/tui/progress/display_props.go [hunk 1]
```diff
@@ -78,7 +78,7 @@ func (m Mode) String() string {
 	var s string
 	switch m {
 	case ModeSpinner:
-		s = "spinner"
+		s = "running"
```
Comment (nexustar) [inline pkg/tui/progress/display_props.go:81]: why, and why not edit other "spinner"

Comment (AstroProfundis) [reply pkg/tui/progress/display_props.go:81]: because they are really spinning...

Comment (AstroProfundis) [reply pkg/tui/progress/display_props.go:81]: ok i think the json marshal could also be adjusted

**Review by nexustar — APPROVED** [review]

**Pull request merged**